lib/rules/no-descending-specificity/README.md

```diff
@@ -31,12 +31,22 @@ This rule only compares rules that are within the same media context. So `a {top
 
 This rule resolves nested selectors before calculating the specificity of the selectors.
 
-## DOM Limitations
+## Limitations
 
-The linter can only check the CSS to check for specificity order. It does not have access to the HTML or DOM in order to interpret the use of the CSS.
+This rule has the following limitations:
+
+- does not have access to HTML or DOM structure
+- does not consider `!important`
+- does not consider individual properties
```

Comment:
```suggestion
This rule doesn't:

- have access to HTML or DOM structure
- consider `!important`
- consider individual properties
```

Minor style nit when writing lists.

Reply:
I agree, this is better, thank you @jeddy3 🙇